glean_parser/translate.py

```diff
@@ -25,39 +25,44 @@
 # Each outputter in the table has the following keys:
 # - "output_func": the main function of the outputter, the one which
 #   does the actual translation.
-# - "clear_output_dir": a flag to clear the target directory before moving there
-#   the generated files.
+# - "extensions": A list of glob patterns on files to clear before generating
```

Comment:
nit: can we rename this to something that hints at file deletion? Like `clear_pattern` (not the best choice) or something similar?